lab_5_scrapper/scrapper.py

```diff
@@ -26,11 +40,28 @@ def _extract_config_content(self) -> ConfigDTO:
         Returns:
             ConfigDTO: Config values
         """
+        config = json.load(self.CRAWLER_CONFIG_PATH.open())
+        config_DTO = ConfigDTO(**config)
+        return config_DTO
 
     def _validate_config_content(self) -> None:
         """
         Ensure configuration parameters are not corrupt.
         """
+        if not all(seed.startswith('https://antropogenez.ru/news/') for seed in self.config_DTO.seed_urls):
+            raise Exception('IncorrectSeedURLError')
+        if self.config_DTO.total_articles < 1 or self.config_DTO.total_articles > 150:
```

Comment:
Please use pre-defined constants from `core_utils` to specify limits

Reply:
![image](https://github.com/fipl-hse/2023-2-level-ctlr/assets/61208744/62ca581c-7c82-40f1-82fb-561f59114a88)

Reply:
done